### ISSUE TITLE
show how far their table has progressed

### ISSUE THREAD

**Issue by ihurrahi** [iamsammak/thehuntgame#257]
up in the air whether we want to show absolute progress (3 out of 8 puzzles done), but we should probably show number of puzzles completed at least

**Comment by cschois1** [comment]
Yeah I think the difficult thing is how to count puzzles A and B. Given that they are "optional", it would be odd to say go from "0 puzzles solved" to "2 puzzles solved" if they solved puzzle 1 before solving puzzle A. Open to ideas on this one though.

**Comment by ihurrahi** [comment]
tabling this for now
